mlir/lib/Analysis/DataFlow/SparseAnalysis.cpp

```diff
@@ -542,7 +554,8 @@ void AbstractSparseBackwardDataFlowAnalysis::visitRegionSuccessors(
     MutableArrayRef<OpOperand> opoperands = operandsToOpOperands(operands);
     ValueRange inputs = successor.getSuccessorInputs();
     for (auto [operand, input] : llvm::zip(opoperands, inputs)) {
-      meet(getLatticeElement(operand.get()), *getLatticeElementFor(op, input));
+      meet(getLatticeElement(operand.get()),
+           *getLatticeElementFor(getProgramPointAfter(op), input));
```

Comment:
@cxy-1993 I looked at this code today. Is this correct? `input` could be a block argument of a region. Is it correct to always use `getProgramPointAfter(op)` or should this distinguish between block argument / op result?